frameworks/elastic/src/main/dist/svc.yml

```diff
@@ -289,9 +289,9 @@ pods:
       - {{SCHEDULER_URI}}
       - {{BOOTSTRAP_URI}}
     tasks:
-      server:
+      node:
```

Comment:
It may make sense to leave `proxylite` as-is, as it isn't really an ES component but instead something we're providing